index.js

```diff
@@ -25,46 +58,38 @@ const skills = [
     'WordPress'
 ];
 
-
 const skillsSection = document.querySelector('#skills');
 const skillsList = skillsSection.querySelector('ul');
 
-
 skills.forEach(skill => {
     const listItem = document.createElement('li');
     listItem.textContent = skill;
     skillsList.appendChild(listItem);
 });
 
-
+// Message Form Section
 const messageForm = document.querySelector('form[name="leave_message"]');
 
 messageForm.addEventListener('submit', function(event) {
-    
     event.preventDefault();
 
-    
     const usersName = event.target.usersName.value;
     const usersEmail = event.target.usersEmail.value;
     const usersMessage = event.target.usersMessage.value;
 
-   
     console.log('Name:', usersName);
     console.log('Email:', usersEmail);
     console.log('Message:', usersMessage);
 
-   
     const messageSection = document.querySelector('#messages');
```

Comment:
To add a message, you need to fix the reference to the 'message' section, as it's returning null. (Hint: There's a typo.) Check your HTML's 'message' section.